src/user_commands/page.rs

```diff
@@ -51,12 +51,13 @@ pub fn UserCommandsPage() -> impl IntoView {
 #[component]
 pub fn CommandSpotlightPage() -> impl IntoView {
     let memo_params_map = use_params_map();
+    let (block_height_sig, _) = create_query_signal::<i64>("q-block-height");
```

Comment:
The (`state_hash`, `txn_hash`) pair will uniquely identify the txn. Using block height may not work in the non-canonical case since there may be more then one block with the same txn hash at the same height.

Reply:
Ok thanks for the detail